generate/unix/Makefile.config

```diff
@@ -227,22 +227,39 @@ CWARNINGFLAGS += \
     -Wold-style-definition\
     -Wno-format-nonliteral\
     -Wredundant-decls
+
+# output directory for tests below
 #
-# Per-host flags and exclusions
+# Taken from the Linux kernel
```

Comment:
The Linux kernel build system is very clearly marked `SPDX-License-Identifier: GPL-2.0-only`